README.md

```diff
@@ -36,18 +36,19 @@ const handlers = __NODE__ && {
 
 export default () => {
   const app = new App(<div></div>);
-
+  // ...
   app.register(RPCToken, RPC);
   app.register(UniversalEventsToken, UniversalEvents);
-  app.configure(RPCHandlersToken, handlers);
-  app.configure(FetchToken, fetch);
-
-  app.register(withDependencies({
-    RPC: RPCToken
-  })(({RPC}) => (ctx, next) => {
-    RPC(ctx).request('getUser', 1).then(console.log) // {some: 'data1'}
+  app.register(RPCHandlersToken, handlers);
+  app.register(FetchToken, fetch);
+
+  app.register(createPlugin({
```

Comment:
I think this example would be simpler if you use the `app.middleware` api

Reply:
Good call, I agree.  I'll update this.